.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: emacs-lisp
+sudo: false
+cache: apt
+env:
+  - EVM_EMACS=emacs-25.1-travis
+  - EVM_EMACS=emacs-25.2-travis
```

Comment:
I typically test my Ruby projects against `ruby`'s latest release(s) as well as the latest Git snapshot because I like to see if stuff works with the new API.  It's okay to omit it, though.